cls/SourceControl/Git/Utils.cls

```diff
@@ -2832,4 +2843,50 @@ ClassMethod SetConfiguredRemote(url) As %String
     quit output
 }
 
+ClassMethod ConfigureFavoriteNamespaces(username As %String, newNamespaces As %String)
+{
+    // Delete all the GIT favorite links for the user
+    set iterator = newNamespaces.%GetIterator()
```

Comment:
Need to use embedded sql here to avoid privilege checking, since %Developer don't have access to the %SYS_Portal.Users table